app/src/main/res/values/strings.xml

```diff
@@ -342,6 +342,8 @@
 
     <string name="text_confirm_exit">Press back again to exit the app</string>
 
+    <string name="text_url_copied_to_clipboard">Copied URL to clipboard</string>
+
```

Comment:
Can we update the other locale files too with this string in `res/values-*`

Reply:
...how would I do that, tho? I don't have translations for it.

Reply:
Just copy paste this same text in all the places, otherwise we'll get runtime crashes when the locale of the device app changes to anything other than english 😏

Reply:
Yikes, I thought it'd fallback to english strings.

Reply:
Done.